frontend/src/components/common/Resource/ResourceTable.tsx

```diff
@@ -41,6 +42,9 @@ function Table(props: ResourceTableProps) {
   const { columns, defaultSortingColumn, ...otherProps } = props;
   const { t } = useTranslation(['glossary', 'frequent']);
   const theme = useTheme();
+  const storeRowsPerPageOptions = useTypedSelector(
+    state => state.ui.settings.tableRowsPerPageOptions
+  );
```

Comment:
Maybe it'd be helpful to have a `useSettings` hook, with a "setting name" argument to make it trigger only if the mentioned setting changed.

Reply:
created a new hook by the name useSettings that does this